cmd/networkcmd/stop.go

```diff
@@ -76,14 +75,20 @@ func stopAndSaveNetwork(flags StopFlags) error {
 
 	if !dontSave {
 		snapshotPath := app.GetSnapshotPath(flags.snapshotName)
-		if err := localnet.TmpNetMigrate(networkDir, snapshotPath); err != nil {
+		if err := localnet.TmpNetMove(networkDir, snapshotPath); err != nil {
 			return err
 		}
 	}
 
-	if err := node.StopLocalNetworkConnectedCluster(app); err != nil {
+	clusterNames, err := localnet.GetRunningLocalClustersConnectedToLocalNetwork(app)
+	if err != nil {
 		return err
 	}
+	for _, clusterName := range clusterNames {
+		if err := localnet.LocalClusterStop(app, clusterName); err != nil {
+			return err
+		}
+	}
```

Comment:
should we continue closing the remaining clusters even if there is an error. Create an []error before calling LocalClusterStop and add to []error if any and just print out the error for the cluster name